### DISCUSSION TITLE
Jellyseerr on unRAID

### DISCUSSION THREAD

**Discussion by theone11** [seerr-team/seerr#806] [Q&A]
I have jellyseer installed on my unraid server in addition to jellyfin.
All is working fine.

I have noticed that sometimes, during "non-working" hour, my unraid disks are spun up and many movie files are accessed.
This started happening when I started using Jellyseerr.

I know Jellyseerr has access to Jellyfin database (defined jellyfin user and API token) so why does it access the actual movie files - Why does it need to do a full library scan?

**Comment by fallenbagel** [comment]
Jellyseerr does not access not need access to physical items. It literally can't because you don't even map them on docker + there's 0 code there that does it. But it does need access to the jellyfin db through api. It also needs access to its own db. These two things will require io from disk

**Reply by theone11** [reply]
Then what is "Jellyfin full library scan"? 

![Screenshot_2024-06-11-11-51-39-619_com.android.chrome-edit.jpg](https://github.com/Fallenbagel/jellyseerr/assets/3348953/d91c093a-c085-49ff-99c3-cd94d405a903)

**Reply by gauthier-th** [reply]
This will get the existing items FROM the Jellyfin API, meaning that Jellyfin will read the content of the media librairies and then send us the list so we can synchronize which media are available. Jellyseerr does not read the files on the disk.

**Reply by theone11** [reply]
Will jellyfin read the library or send the data from its database?

**Reply by gauthier-th** [reply]
IIRC it will read it from its database. The library will be readed when a library scan is runned by Jellyfin.
@Fallenbagel correct me if I'm wrong

**Reply by fallenbagel** [reply]
> IIRC it will read it from its database. The library will be readed when a library scan is runned by Jellyfin.
> @Fallenbagel correct me if I'm wrong

It will read from the jellyfin database. Jellyfin won't perform a library scan. Items are retrieved from the jellyfin api for items retrieval
